pydantic/_internal/_model_construction.py

```diff
@@ -562,9 +559,9 @@ def set_model_fields(
 def complete_model_class(
     cls: type[BaseModel],
     config_wrapper: ConfigWrapper,
+    ns_resolver: NsResolver,
```

Comment:
Let's update the order in the docstring and also where `complete_model_class()` is called.